docs/book/how-to/customize-docker-builds/use-a-prebuilt-image.md

```diff
@@ -0,0 +1,113 @@
+---
+description: "Skip building an image for your ZenML pipeline altogether."
+---
+
+# Use a prebuilt image for pipeline execution
```

Comment:
In general this is missing that the image needs to be stored in a registry where the orchestrator/step operator/other components that require the image can pull from, without any involvement by ZenML.

Reply:
good catch. added it as a warning